emu/src/arm/arm7tdmi.rs

```diff
@@ -32,7 +32,7 @@ impl Default for Arm7tdmi {
     fn default() -> Self {
         let mut s = Self {
             memory: Arc::new(Mutex::new(InternalMemory::default())),
-            cpsr: Psr::from(Mode::User), // FIXME: Starting as User? Not sure
+            cpsr: Psr::from(Mode::Supervisor), // FIXME: Starting as Supervisor? Not sure
```

Comment:
Not sure about this, maybe some bios part will call SWI to switch into Supervisor... Default mode is User. I see other reference starting with System mode 🤔

Reply:
Uhm I don't know, in many places they say that supervisor is used either at reset state or via SWI.

https://www.linuxbaya.com/2019/08/operating-modes-in-arm-7.html
> Supervisor Mode ARM7 enters this mode on reset. It is used to execute the BIOS program (Booting program). This mode can also be invoked by the programmer by SWI (Software Interrupt). 

https://courses.cs.washington.edu/courses/cse466/10au/pdfs/lectures/07-arm_overview.pdf
> Supervisor : entered on reset and when a Software Interrupt
 instruction is executed

Reply:
Let's go for `Supervisor` :)